x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/job_type.tsx

```diff
@@ -17,64 +17,75 @@ interface Props {
-    }
-  );
+interface JobDetails {
+  [key: string]: {
+    helpText: string;
+    icon: string;
+    title: string;
+  };
+}
 
-  const regressionHelpText = i18n.translate(
-    'xpack.ml.dataframe.analytics.create.outlierRegressionHelpText',
-    {
+const jobDetails: JobDetails = {
+  [ANALYSIS_CONFIG_TYPE.OUTLIER_DETECTION]: {
+    helpText: i18n.translate('xpack.ml.dataframe.analytics.create.outlierDetectionHelpText', {
+      defaultMessage: 'Outlier detection identifies unusual data points in the data set.',
+    }),
+    icon: 'outlierDetectionJob',
+    title: 'Outlier detection',
+  },
+  [ANALYSIS_CONFIG_TYPE.REGRESSION]: {
+    helpText: i18n.translate('xpack.ml.dataframe.analytics.create.outlierRegressionHelpText', {
       defaultMessage: 'Regression predicts numerical values in the data set.',
-    }
-  );
-
-  const classificationHelpText = i18n.translate(
-    'xpack.ml.dataframe.analytics.create.classificationHelpText',
-    {
+    }),
+    icon: 'regressionJob',
+    title: 'Regression',
+  },
+  [ANALYSIS_CONFIG_TYPE.CLASSIFICATION]: {
+    helpText: i18n.translate('xpack.ml.dataframe.analytics.create.classificationHelpText', {
       defaultMessage: 'Classification predicts labels of data points in the data set.',
-    }
-  );
+    }),
+    icon: 'classificationJob',
+    title: 'Classification',
+  },
+};
 
-  const helpText = {
-    [ANALYSIS_CONFIG_TYPE.REGRESSION]: regressionHelpText,
-    [ANALYSIS_CONFIG_TYPE.OUTLIER_DETECTION]: outlierHelpText,
-    [ANALYSIS_CONFIG_TYPE.CLASSIFICATION]: classificationHelpText,
-  };
+export const JobType: FC<Props> = ({ type, setFormState }) => {
+  const [selectedCard, setSelectedCard] = useState<any>({});
 
   return (
-    <Fragment>
-      <EuiFormRow
-        fullWidth
-        label={i18n.translate('xpack.ml.dataframe.analytics.create.jobTypeLabel', {
-          defaultMessage: 'Job type',
-        })}
-        helpText={type !== undefined ? helpText[type] : ''}
-      >
-        <EuiSelect
-          fullWidth
-          options={Object.values(ANALYSIS_CONFIG_TYPE).map((jobType) => ({
-            value: jobType,
-            text: jobType.replace(/_/g, ' '),
-            'data-test-subj': `mlAnalyticsCreation-${jobType}-option`,
-          }))}
-          value={type}
-          hasNoInitialSelection={true}
-          onChange={(e) => {
-            const value = e.target.value as AnalyticsJobType;
-            setFormState({
-              previousJobType: type,
-              jobType: value,
-              includes: [],
-              requiredFieldsError: undefined,
-            });
-          }}
-          data-test-subj="mlAnalyticsCreateJobWizardJobTypeSelect"
-        />
-      </EuiFormRow>
-    </Fragment>
+    <>
+      <EuiFlexGroup gutterSize="m" data-test-subj="mlAnalyticsCreateJobWizardJobTypeSelect">
+        {Object.keys(jobDetails).map((jobType) => (
+          <EuiFlexItem key={jobType} grow={false}>
```

Comment:
Using `grow={1}` might be better here (or any alternative which guarantees that the all the cards are the same width). This would guarantee equal width of the three cards. Here the text for each is roughly the same length, so the cards are roughly the same width, but this might not be the case depending on the locale.

![image](https://user-images.githubusercontent.com/7405507/94675711-0e59e880-0312-11eb-86cc-7e4ea075d06d.png)

Reply:
Oh good point - updated in 3c51fbaa5146278c93c81afbd2ca5f007cabb449